app/views/audits/show.html.erb

```diff
@@ -4,6 +4,10 @@
                                         hash_for_host_path(:id => @audit.auditable.to_param).merge(:auth_object => @audit.auditable, :auth_action => 'view'),
                                         :title => _("Host details"),
                                         :class => 'btn btn-info') if @audit.auditable_type == 'Host::Base' && @audit.auditable %>
+<%= title_actions link_to_if_authorized(_("Associated Host"),
+                                        hash_for_host_path(:id => @audit.associated.to_param).merge(:auth_object => @audit.associated, :auth_action => 'view'),
+                                        :title => _("Host details"),
+                                        :class => 'btn btn-info') if @audit.auditable_type.match(/^Nic/) && @audit.associated_type == 'Host::Base' && @audit.associated %>
```

Comment:
I think just `btn-default` (grey instead of dark blue) suffices, clicking on "Associated host" isn't really the main action a user should be doing here, rather just going back to check other audits.

![screenshot from 2017-11-13 11-00-25](https://user-images.githubusercontent.com/598891/32719844-fb55303e-c861-11e7-9724-c385d0ffdd92.png)

cc @Rohoover your thoughts?